make-it/Makefile

```diff
@@ -108,7 +108,7 @@ endif
 .DEFAULT_GOAL=help
 
 $(OUTPUT_DIR):
-	@mkdir -p $(OUTPUT_DIR)
+	"mkdir" -p $(OUTPUT_DIR)
```

Comment:
Please add the @ back, and make a variable MKDIR = "mkdir" -p with a comment explaining that Windows needs the quote, then use that here.